res/openvpn-gui-res-zh-hant.rc

```diff
@@ -169,15 +201,15 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
 BEGIN
     ICON ID_ICO_APP, 0, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v" PACKAGE_VERSION_RESOURCE_STR " - A Windows GUI for OpenVPN\n" \
+    LTEXT "OpenVPN GUI v" PACKAGE_VERSION_RESOURCE_STR " - OpenVPN 的 Windows 圖形化介面\n" \
           "版權所有 (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
           "版權所有 (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n" \
-          "版權所有 (C) 2012-2016 OpenVPN GUI contributors\n" \
+          "版權所有 (C) 2012-2016 OpenVPN GUI 貢獻者\n" \
           "https://github.com/OpenVPN/openvpn-gui/", 0, 36, 15, 206, 42
-    LTEXT "OpenVPN - An application to securely tunnel IP networks " \
-          "over a single UDP port, with support for SSL/TLS-based " \
-          "session Authentication and key exchange, packet " \
-          "encryption, packet authentication, and packet compression.\n" \
+    LTEXT "OpenVPN 是一套可安全地透過單一 UDP 通訊埠穿隧 IP 網路" \
+          "的應用程式。支援以 SSL/TLS 為基礎的連線驗證與金鑰交換" \
```

Comment:
Suggest to replace UDP by TCP/UDP or its equivalent (see the About page in English).